src/main/java/org/cardanofoundation/explorer/api/service/impl/BolnisiMetadataServiceImpl.java

```diff
@@ -199,13 +207,10 @@ public CertDetailsData getCertDetailsData(String txHash, String certNo) {
 
   @Override
   public BolnisiProjectNumberResponse getBolnisiProjectNumber() {
-    BolnisiProjectNumberResponse response = new BolnisiProjectNumberResponse();
-    // Hardcoded values for the number of bottles, wineries, and certificates
-    // TODO
-    response.setNumberOfBottles(BigInteger.valueOf(71683));
-    response.setNumberOfWineries(BigInteger.valueOf(29));
-    response.setNumberOfCertificates(BigInteger.valueOf(1));
-    return response;
+    RestTemplate template = new RestTemplate();
```

Comment:
we have both `restTemplate` and `webClient` any reasons why we need both?

Reply:
Good point, will replace it with webclient.

Reply:
Done